mlflow/store/tracking/rest_store.py

```diff
@@ -313,6 +313,10 @@ def delete_run(self, run_id):
         req_body = message_to_json(DeleteRun(run_id=run_id))
         self._call_endpoint(DeleteRun, req_body)
 
+    def delete_runs(self, run_ids):
+        req_body = message_to_json(DeleteRun(run_id=run_ids))
+        self._call_endpoint(DeleteRun, req_body)
```

Comment:
`DeleteRun` does not support multiple run_ids, you have to add new proto `DeleteRuns`

Reply:
and add a new handler in rest server side for handling `DeleteRuns`